scripts/tools/silabs/ota/ota_multi_image_tool.py

```diff
@@ -163,6 +164,32 @@ def generate_app(args: object):
         return [OTA_APP_TLV_TEMP, args.app_input_file]
 
 
+def generate_wifi_image(args: object):
+    """
+    Generate app payload with descriptor. If a certain option is not specified, use the default values.
+    """
+    logging.info("App descriptor information:")
+
+    descriptor = generate_descriptor(args.app_version, args.app_version_str, args.app_build_date)
```

Comment:
is this required? looks to be application specific

Reply:
Modified to wifi-ta version instead of app version